### PR TITLE
OCPBUGS-5461: Add ironic IP to no_proxy

### PR DESCRIPTION
**Pull request by derekhiggins**
Ensure internal ironic traffic bypasses any proxies

### CODE REVIEW

**Comment by elfosardo** [comment]
@derekhiggins looks good, thanks! Can you please add this also to metal3-io?

/lgtm

File: scripts/configure-ironic.sh [hunk 1]
```diff
@@ -97,3 +97,6 @@ configure_client_basic_auth() {
 
 configure_client_basic_auth ironic-inspector
 configure_client_basic_auth ironic-rpc
+
+# Make sure ironic traffic bypasses any proxies
+export NO_PROXY=$NO_PROXY,$IRONIC_IP
```
Comment (dtantsur) [inline scripts/configure-ironic.sh:102]: Is NO_PROXY defined by default? Seems like no, so you may need

```
export NO_PROXY="${NO_PROXY:-},$IRONIC_IP"
```

**Comment by derekhiggins** [comment]
/hold
new version incomming

**Comment by derekhiggins** [comment]
upstream version https://github.com/metal3-io/ironic-image/pull/405

**Comment by derekhiggins** [comment]
/unhold
upstream merged

**Pull request merged**